people/methods/find.js

```diff
@@ -0,0 +1,29 @@
+
+var mysql = require('mysql');
+
+const con = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "123456",
+  database: "interns"
+});
+
+con.connect(function (err) {
```

Comment:
why are you creating a connection every time when you call the find method

you should only create it once in the lifecycle of the application check index.js of the sample application